<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adjust</class>
 <widget class="QWidget" name="Adjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="planeAppearLabel">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>61</y>
     <width>144</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plane Appear Probability</string>
   </property>
  </widget>
  <widget class="QSlider" name="PlaneAppearSlider">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>61</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="AttackProbabilityLabel">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>111</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack Probability</string>
   </property>
  </widget>
  <widget class="QSlider" name="AttackProbabilitySlider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="SupplyProbabilityLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Supply Probability</string>
   </property>
  </widget>
  <widget class="QSlider" name="SupplyProbabilitySlider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
